product_customer_code/__openerp__.py

```diff
@@ -1,35 +1,14 @@
 # coding: utf-8
-###########################################################################
-#    Module Writen to OpenERP, Open Source Management Solution
-#
-#    Copyright (c) 2012 Vauxoo - http://www.vauxoo.com/
-#    All Rights Reserved.
-#    info Vauxoo (info@vauxoo.com)
-############################################################################
-#    Coded by: el_rodo_1 (rodo@vauxoo.com)
-############################################################################
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2016 Vauxoo (https://www.vauxoo.com) info@vauxoo.com
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
 {
     "name": "Products Customer Code",
-    "version": "8.0.0.1.6",
+    "version": "8.0.0.1.7",
     "author": "Vauxoo",
     "category": "Generic Modules/Product",
     "website": "http://www.vauxoo.com/",
-    "license": "AGPL-3",
+    "license": "LGPL-3",
```

Comment:
Is this correct?
I mean, I'm not sure of change a public module from `AGPL` to `LGPL`